packages/base-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- **BREAKING:** Change the types for `BaseControllerV1` class fields `initialConfig`, `initialState` from `C`, `S` to `Partial<C>`, `Partial<S>` ([#3959](https://github.com/MetaMask/core/pull/3959))
```

Comment:
It looks like these are private properties, so is this breaking?

Reply:
The constructor options `config`, `state` are handled slightly differently since there's no type assertion for the default parameter, but this probably doesn't warrant a changelog entry much less as a breaking change. Removed: https://github.com/MetaMask/core/pull/3959/commits/0a6698764b016a3ee87a4a45d4b6460360cbba43